pages/data-design/global-namespace.mdx

```diff
@@ -1,68 +1,72 @@
----
-layout: 'guide'
-title: 'Global Namespace'
-abstract: 'Global namespace for events and properties'
----
-import Link from '../../components/Link';
+import { Callout } from 'nextra/components';
 
+# Global Namespace
+
+_Global namespace for events and properties_
 
 This guide is a part of our Avo Design Decision series, and it covers our decision for this:
 
-> 👉 Avo has a **global [namespace](https://en.wikipedia.org/wiki/Namespace)** to ensure that **all events and properties in the tracking plan have unique names** that they can be easily identified.
+<Callout emoji="👉">
+  Avo has a **global [namespace](https://en.wikipedia.org/wiki/Namespace)** to
+  ensure that **all events and properties in the tracking plan have unique
+  names** that they can be easily identified.
+</Callout>
 
-Data design is a rising skill. Please check out our guide on <Link passHref href="/data-design/naming-conventions"><a>Naming conventions of events and properties</a></Link>.
+Data design is a rising skill. Please check out our guide on [Naming conventions of events and properties](/data-design/naming-conventions).
 
-Naming inconsistencies often arise either during the "data design" step, not only implementation step. This is particularly true as product organizations scale up and multiple teams contribute to the tracking plan. (It's difficult enough to remember all the events you *yourself* created, let alone be aware of all the events other people have created).
+Naming inconsistencies often arise either during the "data design" step, not only implementation step. This is particularly true as product organizations scale up and multiple teams contribute to the tracking plan. (It's difficult enough to remember all the events you _yourself_ created, let alone be aware of all the events other people have created).
 
-Having a global namespace for all events and properties allows Avo to support people in various ways in designing better data. 
+Having a global namespace for all events and properties allows Avo to support people in various ways in designing better data.
 
 For example:
 
-1. Avo **prevents creating a duplicate event** whose only difference from an existing event is the *casing*. Same for property names. 
-
-    For example: Avo won't allow you create a `signupStart` event if `SIGNUP_START` already exists:
+1. Avo **prevents creating a duplicate event** whose only difference from an existing event is the _casing_. Same for property names.
 
-    <img
-	  src={require('../../images/best-practices/naming-conventions/already_exists.png')}
-	  alt="Property already exists"
-	  width="75%"
-	/>
+   For example: Avo won't allow you create a `signupStart` event if `SIGNUP_START` already exists:
 
-2. Avo ***nudges* you if you try to create an event in a casing that contradicts** your tracking plan casing convention. Same for property names. 
+   <img
+     src={'/images/best-practices/naming-conventions/already_exists.png'}
```

Comment:
What would it take for us to use `next/image`? Can it accept percentages? 🤔

Reply:
I don't think so. We need the actual width and height of the image so it can optimise the file delivery